variables.tf

```diff
@@ -410,6 +410,30 @@ variable "alb_ingress_healthcheck_protocol" {
   description = "The protocol to use to connect with the target. Defaults to `HTTP`. Not applicable when `target_type` is `lambda`"
 }
 
+variable "alb_ingress_health_check_healthy_threshold" {
+  type        = number
+  default     = 2
+  description = "The number of consecutive health checks successes required before healthy"
+}
+
+variable "alb_ingress_health_check_unhealthy_threshold" {
+  type        = number
+  default     = 2
+  description = "The number of consecutive health check failures required before unhealthy"
+}
+
+variable "alb_ingress_health_check_interval" {
+  type        = number
+  default     = 15
+  description = "The duration in seconds in between health checks"
+}
+
+variable "alb_ingress_health_check_timeout" {
+  type        = number
+  default     = 10
+  description = "The amount of time to wait in seconds before failing a health check request"
+}
```

Comment:
I confirmed that these are the same defaults in the upstream module

 https://github.com/cloudposse/terraform-aws-alb-ingress